apps/frontend-manage/src/components/resources/answerCollections/AnswerCollectionItem.tsx

```diff
@@ -59,6 +71,9 @@ function AnswerCollectionItem({
     setDeletionModal,
   })
 
+  // Add activity log action to the beginning of the dropdown items
+  const allDropdownItems = [activityLogAction, ...dropdownItems]
```

Comment:
Similar as for the live quiz actions, we should probably integrate the action properly into the dropdown hook instead of adding it in this way?